<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_f9i8yvicf7/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -17,10 +17,11 @@ source:
     - patches/fix_mac10_9_clock_realtime.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - m2-bison         # [win]
     - m2-diffutils     # [win]
@@ -35,19 +36,19 @@ requirements:
     - patch            # [unix]
     - bison            # [linux]
   host:
-    - krb5 1.20.1
+    - krb5 {{ krb5 }}
     - openssl {{ openssl }}
-    - readline 8.1.2         # [not win]
+    - readline {{ readline }}  # [not win]
     - icu {{ icu }}
-    - libuuid 1.41.5         # [linux]
+    - libuuid 1.41.5           # [linux]
     - libxml2 {{ libxml2 }}
-    - libxslt {{ libxslt }}  # [linux]
-    - lz4-c 1.9.4
-    - openldap 2.6.4         # [not (win or s390x)]
-    - tzdata                 # [linux]
+    - libxslt {{ libxslt }}    # [linux]
+    - lz4-c {{ lz4_c }}
+    - openldap {{ openldap }}  # [not (win or s390x)]
+    - tzdata                   # [linux]
     - zstd {{ zstd }}
     - zlib {{ zlib }}
-    - msinttypes r26         # [win and vc<14]
+    - msinttypes r26           # [win and vc<14]
 
 outputs:
   - name: postgresql
@@ -59,22 +60,23 @@ outputs:
     requirements:
       build:
         # solely for sake of lining up vc versions and other runtimes
+        - {{ stdlib('c') }}
         - {{ compiler('c') }}
         - meson            # [win]
         - perl
         - make             # [unix]
       host:
         # these are here for sake of run_exports taking effect
-        - krb5 1.20.1
-        - openldap 2.6.4   # [not (win or s390x)]
+        - krb5 {{ krb5 }}
+        - openldap {{ openldap }}  # [not (win or s390x)]
         - openssl {{ openssl }}
-        - readline 8.1.2   # [not win]
+        - readline {{ readline }}  # [not win]
         - icu {{ icu }}
         - libxml2 {{ libxml2 }}
-        - lz4-c 1.9.4
+        - lz4-c {{ lz4_c }}
         - zstd {{ zstd }}
         - zlib {{ zlib }}
-        - msinttypes r26   # [win and vc<14]
+        - msinttypes r26           # [win and vc<14]
         - {{ pin_subpackage('libpq', exact=True) }}
       run:
         - {{ pin_subpackage('libpq', exact=True) }}
@@ -114,17 +116,18 @@ outputs:
     requirements:
       build:
         # solely for sake of lining up vc versions and other runtimes
+        - {{ stdlib('c') }}
         - {{ compiler('c') }}
         - meson  # [win]
         - perl
         - make   # [unix]
       host:
         # these are here for sake of run_exports taking effect
-        - krb5 1.20.1
-        - openldap 2.6.4   # [not (win or s390x)]
+        - krb5 {{ krb5 }}
+        - openldap {{ openldap }}  # [not (win or s390x)]
         - openssl {{ openssl }}
-        - zlib {{ zlib }}  # [win]
-        - msinttypes r26   # [win and vc<14]
+        - zlib {{ zlib }}          # [win]
+        - msinttypes r26           # [win and vc<14]
       run:
         # Versions constraints come from run_exports
         - krb5

